.github/workflows/validate-windows-ut.yml

```diff
@@ -12,7 +12,13 @@ jobs:
         shell: powershell
         run: |
           Invoke-Pester parts/windows/*.tests.ps1 -Passthru
-      - name: Perform a Pester test from the staging/cse/windows/*.tests.ps1 file
-        shell: powershell
-        run: |
-          Invoke-Pester staging/cse/windows/*.tests.ps1 -Passthru
\ No newline at end of file
+  pester-test-2:
```

Comment:
What is the purpose to separate them into two stages?

Reply:
To avoid the case that we meet before: the failure in the first run will not be shown due toe the succeed in the second run.

Reply:
I think I get something mixed here. Change it back and test again.